bin/release

```diff
@@ -26,10 +23,10 @@ infer_release_type() {
 setup() {
     # Ensure we have exactly one annotated tag pointing at HEAD
     HEAD_TAGS=$(git tag --points-at HEAD)
-	TAG_COUNT=$(echo $(echo $HEAD_TAGS | wc -w)) # mac hack
+    TAG_COUNT=$(echo "$HEAD_TAGS" | wc -w) # mac hack
```

Comment:
https://github.com/weaveworks/scope/commit/58a1528ba6f22ae9a7e84ee1eb22287e987734dd#commitcomment-20505530

We probably need to add a comment to tell shellcheck not to remove the double echo then...